packs/elastic/actions/search.py

```diff
@@ -0,0 +1,87 @@
+from easydict import EasyDict
+from lib.items_selector import ItemsSelector
+from lib.elastic_action import ElasticAction
+from collections import defaultdict
+import logging
+import sys
+import elasticsearch
+import json
+
+logger = logging.getLogger(__name__)
+
+
+class SearchRunner(ElasticAction):
```

Comment:
Minor naming thing - I think `FindDocumentsAction` or similar is better since it's more obvious what the class / action does.